javascript/src/main/components/BasicCourseSearch/BasicCourseTable.js

```diff
@@ -2,22 +2,78 @@ import React from "react";
 import BootstrapTable from 'react-bootstrap-table-next';
 
 const BasicCourseTable = ( {classes} ) => {
-  
+
+  const sections = [];
+  classes.forEach(
+    (course) => {
+      course.classSections.forEach(
+        (section) => {
+          section.course =
+          {
+          courseId: course.courseId,
+          title: course.title,
+          unitsFixed: course.unitsFixed
+          };
+         sections.push(section);
+       }
+      )
+    }
+  );
+const rowStyle = (row, rowIndex) => {
+  console.log(`row=`,row, `rowIndex`, rowIndex);
```

Comment:
I wonder whether we still need this? Unless we're logging something that should be seen even in production, we should probably remove this.

Reply:
Which line are you saying to remove for this specific PR?